src/main/java/dk/kb/present/View.java

```diff
@@ -187,11 +188,28 @@ private String getFirstPresentationManifestation(DsRecordDto record) {
         // TODO: Figure how to choose correct manifestation for record, if more than one is present
         // Return first child record of manifestation type = 2. If there are multiple presentation manifestations,
         // the rest are currently not added to the transformation
-        return record.getChildren() == null ? "" :
+        List<String> presentationManifestations = record.getChildren() == null ? Collections.singletonList("") :
                 record.getChildren().stream()
                         .map(this::getNonNullChild)
                         .filter(childData -> childData.contains("<ManifestationRelRef>2</ManifestationRelRef>"))
-                        .findFirst().orElse("");
+                        .collect(Collectors.toList());
+
+        return returnPresentationManifestationFromList(presentationManifestations, record.getId());
+    }
+
+    /**
+     * Returns the first presentation manifestation for a record. If there are more than one presentation manifestation
+     * for the record a warning will be logged.
+     * @param presentationManifestations list of all presentation manifestations for a record
+     * @return the first presentation manifestation for a record.
+     */
+    private String returnPresentationManifestationFromList(List<String> presentationManifestations, String recordId) {
+        if (presentationManifestations.size() > 1) {
+            log.warn("Multiple presentation manifestations were present for record with id: '{}'. " +
+                    "Only the first has been returned", recordId);
+        }
+
+        return presentationManifestations.get(0);
     }
```

Comment:
I like this separation as it is also a preparation for later code which might select the manifestation using a more elaborate schema than "Meh. Just pick the first one".